### DISCUSSION TITLE
姑且问问有没有兴趣写 Tauri...

### DISCUSSION THREAD

**Discussion by BTMuli** [Arondight/Adachi-BOT#996] [Q&A]
rt，[Tauri.Genshin](https://github.com/BTMuli/Tauri.Genshin)，~~恬不知耻的来拉劳力了~~

**Comment by mark9804** [comment]
Tauri 是那个直接调用系统 webview 的框架？

**Reply by BTMuli** [reply]
嗯，比electron轻量，就是后端用rust写的不太会改

**Reply by mark9804** [reply]
可以先写一个前端 demo，然后带着 demo 去论坛发招人帖

demo 可以写的烂但一定要有，我不止听到过一次“当初我看到你有 demo 才来的，不然我才不来”

**Reply by BTMuli** [reply]
> 可以先写一个前端 demo，然后带着 demo 去论坛发招人帖
> 
> demo 可以写的烂但一定要有，我不止听到过一次“当初我看到你有 demo 才来的，不然我才不来”

demo倒是有，目前写的几个小功能点都打包了，不过论坛有没有推荐的，给指指路

**Reply by mark9804** [reply]
NGA和B站都可以，按我身边统计学NGA写rust的人会稍微多一点

**Comment by Arondight** [comment]
彻底没时间做这些，这个项目都已经凉了很久了……

**Reply by mark9804** [reply]
老大哥最近是在忙工作还是什么的，我看我这边几个前后端和 UX 都在跑路或者准备跑路，感觉互联网行业有点微妙了

**Reply by Arondight** [reply]
> 老大哥最近是在忙工作还是什么的，我看我这边几个前后端和 UX 都在跑路或者准备跑路，感觉互联网行业有点微妙了

忙工作，我这儿比较稳定，不过也没啥钱，互联网形势我不清楚，已经好多年没接触过互联网了，今年经济很不好，全球都在裁员，特别是互联网，这里的资本无情又没有顾虑
